iothub/device/src/InternalClient.cs

```diff
@@ -1543,20 +1543,7 @@ public Task UploadToBlobAsync(string blobName, Stream source, CancellationToken
                     throw Fx.Exception.Argument(nameof(blobName), "Path segment count cannot exceed 254");
                 }
 
-                var context = new PipelineContext();
-                context.Set(_productInfo);
-
-                var transportSettings = new Http1TransportSettings();
```

Comment:
The difference between this one and the ctor one, is that the ctor could be any other transport, and this is specifically Http.

Reply:
file upload is always over HTTP though

Reply:
Never mind.

Reply:
including the one in the ctor.